python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -738,6 +739,23 @@ def _get_exposure_info(
 
             for row in visitSummary:
                 detector = row["id"]
+
+                wcs = row.getWcs()
+                if wcs is None:
+                    self.log.warning(
+                        "WCS is None for visit %d, detector %d: this extension will be dropped.",
+                        visit,
+                        detector,
+                    )
+                    continue
```

Comment:
Should this be a warning, or an info-level message? Also, saying "extension" in the log is unhelpful for anyone who doesn't know the gbdes internals.

Reply:
I think a warning is right (from the dev-guide: "WARNING: for conditions that may indicate a problem but that allow continued execution"). I think there should be a note of caution for the resulting WCS. 
I changed the message to say "...this extension (visit/detector) will be dropped."

Reply:
Do you have a way to flag the WCSs that are "reconstructed" in this case in the output, to warn users? A warning message may be appropriate here, but it's not sufficient to tell people that the output WCS is potentially unreliable.

Reply:
No, neither the WCSs nor the visitSummary tables have a flag for WCS quality. The WCS fit will have higher uncertainty, as it would if there were very few stars to fit, and Eli and I have talked a bit about how to propagate this information.